_includes/footer.html

```diff
@@ -56,13 +56,6 @@ <h4 style="color:#eee;">Für was wir stehen</h4>
 
     <div class='large-3 medium-3 columns'>
       <div class='spacing'></div>
-      <ul>
```

Comment:
I'm for keeping the contact footer, but I do not know how accurate the information is in there.

Reply:
@simonamoise here, too :)